orchestra/workflow/load.py

```diff
@@ -206,3 +210,15 @@ def _set_step_dependencies(step, step_data, dependency_attr, dependency_model,
             '{}.{} contains a non-existent slug.'
             .format(step_data['slug'], dependency_attr))
     getattr(step, dependency_attr).set(dependencies)
+
+
+def _set_step_templates(step, step_data, template_attr, template_model,
```

Comment:
this looks a lot like _set_step_dependencies. can you call that instead?

Reply:
Yes! The naming is wrong if we do that- so I'll rename the function also.